force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -11813,6 +11837,54 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
         <value>A paid Payment must have a Payment Date.</value>
     </labels>
+    <labels>
+        <fullName>pmtRefundPaymentErrorMessage</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
+        <value>There was an error processing the refund.</value>
```

Comment:
@andrewyu-salesforce - The description for this label indicates that a Paid Payment Date is missing. Is that the reason for this error, or is this error meant to cover any processing error?

Reply:
Sorry, I forgot to update the description. This is the error title that will be displayed with all other error messages in the widget.

Reply:
Ok, thanks. I think the label is fine as is then.